index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Sign-Up Form</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/06aa7828ae.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+    <main>
+        <header>
+            <div class="account">
+                <div class="acc">Already have an Inuit account? <br> <a class="acc-link" href="#">I forgot my User ID or
+                        Password</a></div>
+
+                <div class="ol"><i class="fas fa-lock"></i> Sign In</div>
+            </div>
+        </header>
+        <section>
+            <div class="logo"><img src="image/logo.png" alt="logo"></div>
```

Comment:
- [ ] This should be divided by three, with each part with the respective logo and text. 

![image](https://user-images.githubusercontent.com/61479648/93826447-a1718f00-fc3d-11ea-86ab-61d6c766359c.png)